src/client/Microsoft.Identity.Client/Platforms/Features/OpenTelemetry/OtelInstrumentation.cs

```diff
@@ -125,8 +125,9 @@ public void LogSuccessMetrics(
                         new(TelemetryConstants.Platform, platform),
                         new(TelemetryConstants.ApiId, apiId),
                         new(TelemetryConstants.TokenSource, authResultMetadata.TokenSource),
-                        new(TelemetryConstants.CacheLevel, cacheLevel), 
-                        new(TelemetryConstants.CacheRefreshReason, authResultMetadata.CacheRefreshReason)); 
+                        new(TelemetryConstants.CacheLevel, cacheLevel),
+                        new(TelemetryConstants.CacheRefreshReason, authResultMetadata.CacheRefreshReason),
+                        new(TelemetryConstants.TokenType, authResultMetadata.TelemetryTokenType));
```

Comment:
Added the tokenType to the other histograms as well. Do we want to be able to differentiate between different token types for all Histograms? Or is this to far?
@neha-bhargava @bgavrilMS

Reply:
Yeah, I think we should add it to MsalTotalDuration and to MsalDurationInHttp

Reply:
+1 to adding it to MsalTotalDuration and MsalDurationInHttp. This way we can distinguish latencies for different token types. I don't think the cache latency should be different based on token types.